co2eq/transportation.js

```diff
@@ -112,6 +113,7 @@ export function carbonEmissions(activity) {
 
   // Take into account the passenger count if this is a car or motorbike
   if (activity.transportationMode === TRANSPORTATION_MODE_CAR || activity.transportationMode === TRANSPORTATION_MODE_MOTORBIKE) {
+    // Take into account if the input for the car was detailed (with brand and model)
```

Comment:
No need to modify this file (we'll make sure to call your model when appropriate).
Can you revert this file?